 [INFO ] 2023-05-14 10:19:31.012 [restartedMain] SpringBootRndApplication - Starting SpringBootRndApplication using Java 11.0.17 on tigerit-HP-ProDesk-600-G5-PCI-MT with PID 6523 (/home/tigerit/home_office/spring-boot-rnd/target/classes started by tigerit in /home/tigerit/home_office/spring-boot-rnd)
[DEBUG] 2023-05-14 10:19:31.013 [restartedMain] SpringBootRndApplication - Running with Spring Boot v2.7.0, Spring v5.3.20
[INFO ] 2023-05-14 10:19:31.013 [restartedMain] SpringBootRndApplication - No active profile set, falling back to 1 default profile: "default"
[DEBUG] 2023-05-14 10:19:31.980 [restartedMain] HikariConfig - HikariPool-1 - configuration:
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - allowPoolSuspension................................false
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - autoCommit................................true
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - catalog................................none
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - connectionInitSql................................none
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - connectionTestQuery................................none
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - connectionTimeout................................30000
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - dataSource................................none
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - dataSourceClassName................................none
[DEBUG] 2023-05-14 10:19:31.981 [restartedMain] HikariConfig - dataSourceJNDI................................none
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - dataSourceProperties................................{password=<masked>}
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - driverClassName................................none
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - exceptionOverrideClassName................................none
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - healthCheckProperties................................{}
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - healthCheckRegistry................................none
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - idleTimeout................................600000
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - initializationFailTimeout................................1
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - isolateInternalQueries................................false
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - jdbcUrl................................jdbc:postgresql://tiny.db.elephantsql.com:5432/rzumhkcr
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - keepaliveTime................................0
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - leakDetectionThreshold................................0
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - maxLifetime................................1800000
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - maximumPoolSize................................10
[DEBUG] 2023-05-14 10:19:31.982 [restartedMain] HikariConfig - metricRegistry................................none
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - metricsTrackerFactory................................none
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - minimumIdle................................10
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - password................................<masked>
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - poolName................................"HikariPool-1"
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - readOnly................................false
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - registerMbeans................................false
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - scheduledExecutor................................none
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - schema................................none
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - threadFactory................................internal
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - transactionIsolation................................default
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - username................................"rzumhkcr"
[DEBUG] 2023-05-14 10:19:31.983 [restartedMain] HikariConfig - validationTimeout................................5000
[INFO ] 2023-05-14 10:19:31.983 [restartedMain] HikariDataSource - HikariPool-1 - Starting...
[DEBUG] 2023-05-14 10:19:31.988 [restartedMain] DriverDataSource - Loaded driver with class name org.postgresql.Driver for jdbcUrl=jdbc:postgresql://tiny.db.elephantsql.com:5432/rzumhkcr
[DEBUG] 2023-05-14 10:19:33.278 [restartedMain] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@630aca3d
[INFO ] 2023-05-14 10:19:33.282 [restartedMain] HikariDataSource - HikariPool-1 - Start completed.
[DEBUG] 2023-05-14 10:19:33.382 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=1, active=1, idle=0, waiting=0)
[DEBUG] 2023-05-14 10:19:34.333 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@4b12b0e7
[DEBUG] 2023-05-14 10:19:35.208 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@45d547f5
[DEBUG] 2023-05-14 10:19:35.748 [restartedMain] JwtRequestFilter - Filter 'jwtRequestFilter' configured for use
[DEBUG] 2023-05-14 10:19:36.143 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@5aa3403c
[DEBUG] 2023-05-14 10:19:36.987 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@35264e72
[INFO ] 2023-05-14 10:19:37.108 [restartedMain] SpringBootRndApplication - Started SpringBootRndApplication in 6.29 seconds (JVM running for 6.815)
[DEBUG] 2023-05-14 10:19:37.502 [HikariPool-1 connection adder] PoolBase - HikariPool-1 - Failed to create/setup connection: FATAL: too many connections for role "rzumhkcr"
[DEBUG] 2023-05-14 10:19:37.502 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:38.293 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:39.240 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:40.372 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:41.766 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:43.582 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:46.059 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:49.491 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:19:54.351 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:20:01.332 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:20:03.384 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=5, active=0, idle=5, waiting=0)
[INFO ] 2023-05-14 10:20:04.561 [http-nio-8081-exec-1] JwtRequestFilter - not applying custom filter to: /
[INFO ] 2023-05-14 10:20:04.685 [http-nio-8081-exec-2] JwtRequestFilter - not applying custom filter to: /v3/api-docs/swagger-config
[INFO ] 2023-05-14 10:20:04.690 [http-nio-8081-exec-3] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[WARN ] 2023-05-14 10:20:04.690 [http-nio-8081-exec-3] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:20:04.690 [http-nio-8081-exec-3] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:20:04.696 [http-nio-8081-exec-3] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:20:04.696 [http-nio-8081-exec-3] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:20:04.705 [http-nio-8081-exec-4] JwtRequestFilter - not applying custom filter to: /v3/api-docs
[INFO ] 2023-05-14 10:20:04.726 [http-nio-8081-exec-5] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-16x16.png
[WARN ] 2023-05-14 10:20:04.726 [http-nio-8081-exec-5] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:20:04.726 [http-nio-8081-exec-5] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:20:04.726 [http-nio-8081-exec-5] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:20:04.726 [http-nio-8081-exec-5] JwtRequestFilter - exceptionOccurred true
[DEBUG] 2023-05-14 10:20:11.547 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[INFO ] 2023-05-14 10:20:12.044 [http-nio-8081-exec-6] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[WARN ] 2023-05-14 10:20:12.044 [http-nio-8081-exec-6] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:20:12.044 [http-nio-8081-exec-6] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:20:12.044 [http-nio-8081-exec-6] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:20:12.044 [http-nio-8081-exec-6] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:20:12.084 [http-nio-8081-exec-7] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-16x16.png
[WARN ] 2023-05-14 10:20:12.084 [http-nio-8081-exec-7] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:20:12.084 [http-nio-8081-exec-7] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:20:12.084 [http-nio-8081-exec-7] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:20:12.084 [http-nio-8081-exec-7] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:20:14.673 [http-nio-8081-exec-8] JwtRequestFilter - not applying custom filter to: /health/status
[DEBUG] 2023-05-14 10:20:22.132 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[INFO ] 2023-05-14 10:22:49.254 [restartedMain] SpringBootRndApplication - Starting SpringBootRndApplication using Java 11.0.17 on tigerit-HP-ProDesk-600-G5-PCI-MT with PID 7140 (/home/tigerit/home_office/spring-boot-rnd/target/classes started by tigerit in /home/tigerit/home_office/spring-boot-rnd)
[DEBUG] 2023-05-14 10:22:49.255 [restartedMain] SpringBootRndApplication - Running with Spring Boot v2.7.0, Spring v5.3.20
[INFO ] 2023-05-14 10:22:49.256 [restartedMain] SpringBootRndApplication - No active profile set, falling back to 1 default profile: "default"
[DEBUG] 2023-05-14 10:22:50.237 [restartedMain] HikariConfig - HikariPool-1 - configuration:
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - allowPoolSuspension................................false
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - autoCommit................................true
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - catalog................................none
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - connectionInitSql................................none
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - connectionTestQuery................................none
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - connectionTimeout................................30000
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - dataSource................................none
[DEBUG] 2023-05-14 10:22:50.238 [restartedMain] HikariConfig - dataSourceClassName................................none
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - dataSourceJNDI................................none
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - dataSourceProperties................................{password=<masked>}
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - driverClassName................................none
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - exceptionOverrideClassName................................none
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - healthCheckProperties................................{}
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - healthCheckRegistry................................none
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - idleTimeout................................600000
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - initializationFailTimeout................................1
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - isolateInternalQueries................................false
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - jdbcUrl................................jdbc:postgresql://tiny.db.elephantsql.com:5432/rzumhkcr
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - keepaliveTime................................0
[DEBUG] 2023-05-14 10:22:50.239 [restartedMain] HikariConfig - leakDetectionThreshold................................0
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - maxLifetime................................1800000
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - maximumPoolSize................................10
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - metricRegistry................................none
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - metricsTrackerFactory................................none
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - minimumIdle................................10
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - password................................<masked>
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - poolName................................"HikariPool-1"
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - readOnly................................false
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - registerMbeans................................false
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - scheduledExecutor................................none
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - schema................................none
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - threadFactory................................internal
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - transactionIsolation................................default
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - username................................"rzumhkcr"
[DEBUG] 2023-05-14 10:22:50.240 [restartedMain] HikariConfig - validationTimeout................................5000
[INFO ] 2023-05-14 10:22:50.240 [restartedMain] HikariDataSource - HikariPool-1 - Starting...
[DEBUG] 2023-05-14 10:22:50.246 [restartedMain] DriverDataSource - Loaded driver with class name org.postgresql.Driver for jdbcUrl=jdbc:postgresql://tiny.db.elephantsql.com:5432/rzumhkcr
[DEBUG] 2023-05-14 10:22:51.640 [restartedMain] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@4c483f7c
[INFO ] 2023-05-14 10:22:51.643 [restartedMain] HikariDataSource - HikariPool-1 - Start completed.
[DEBUG] 2023-05-14 10:22:51.744 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=1, active=1, idle=0, waiting=0)
[DEBUG] 2023-05-14 10:22:52.684 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@1a75cf94
[DEBUG] 2023-05-14 10:22:53.655 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@14b222aa
[DEBUG] 2023-05-14 10:22:54.285 [restartedMain] JwtRequestFilter - Filter 'jwtRequestFilter' configured for use
[DEBUG] 2023-05-14 10:22:54.561 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@61b244db
[DEBUG] 2023-05-14 10:22:55.476 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@64909d35
[INFO ] 2023-05-14 10:22:55.561 [restartedMain] SpringBootRndApplication - Started SpringBootRndApplication in 6.497 seconds (JVM running for 7.028)
[DEBUG] 2023-05-14 10:22:56.019 [HikariPool-1 connection adder] PoolBase - HikariPool-1 - Failed to create/setup connection: FATAL: too many connections for role "rzumhkcr"
[DEBUG] 2023-05-14 10:22:56.020 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:22:56.841 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:22:57.770 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:22:58.894 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:00.308 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[INFO ] 2023-05-14 10:23:00.464 [http-nio-8081-exec-1] JwtRequestFilter - doFilterInternal path: /swagger-ui/index.html
[WARN ] 2023-05-14 10:23:00.464 [http-nio-8081-exec-1] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:23:00.464 [http-nio-8081-exec-1] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:23:00.470 [http-nio-8081-exec-1] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:23:00.470 [http-nio-8081-exec-1] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:23:00.560 [http-nio-8081-exec-2] JwtRequestFilter - not applying custom filter to: /v3/api-docs/swagger-config
[INFO ] 2023-05-14 10:23:00.565 [http-nio-8081-exec-3] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[WARN ] 2023-05-14 10:23:00.565 [http-nio-8081-exec-3] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:23:00.565 [http-nio-8081-exec-3] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:23:00.565 [http-nio-8081-exec-3] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:23:00.565 [http-nio-8081-exec-3] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:23:00.590 [http-nio-8081-exec-4] JwtRequestFilter - not applying custom filter to: /v3/api-docs
[INFO ] 2023-05-14 10:23:00.602 [http-nio-8081-exec-5] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[WARN ] 2023-05-14 10:23:00.602 [http-nio-8081-exec-5] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:23:00.602 [http-nio-8081-exec-5] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:23:00.602 [http-nio-8081-exec-5] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:23:00.602 [http-nio-8081-exec-5] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:23:00.606 [http-nio-8081-exec-6] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-16x16.png
[WARN ] 2023-05-14 10:23:00.606 [http-nio-8081-exec-6] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:23:00.606 [http-nio-8081-exec-6] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:23:00.606 [http-nio-8081-exec-6] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:23:00.606 [http-nio-8081-exec-6] JwtRequestFilter - exceptionOccurred true
[DEBUG] 2023-05-14 10:23:02.125 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:04.540 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:07.976 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:12.753 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:19.707 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:21.746 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=5, active=0, idle=5, waiting=0)
[DEBUG] 2023-05-14 10:23:29.958 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:40.519 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:51.051 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:23:51.746 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=5, active=0, idle=5, waiting=0)
[DEBUG] 2023-05-14 10:23:51.746 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool is at sufficient level.
[DEBUG] 2023-05-14 10:24:01.700 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:24:12.233 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:24:21.747 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=5, active=0, idle=5, waiting=0)
[DEBUG] 2023-05-14 10:24:21.747 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool is at sufficient level.
[DEBUG] 2023-05-14 10:24:22.811 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:24:33.411 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:24:43.996 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:24:51.747 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=5, active=0, idle=5, waiting=0)
[DEBUG] 2023-05-14 10:24:51.748 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Fill pool skipped, pool is at sufficient level.
[DEBUG] 2023-05-14 10:24:54.587 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:25:05.233 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[INFO ] 2023-05-14 10:25:13.581 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown initiated...
[DEBUG] 2023-05-14 10:25:13.582 [SpringApplicationShutdownHook] HikariPool - HikariPool-1 - Before shutdown stats (total=5, active=0, idle=5, waiting=0)
[DEBUG] 2023-05-14 10:25:13.619 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@4c483f7c: (connection evicted)
[DEBUG] 2023-05-14 10:25:13.714 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@1a75cf94: (connection evicted)
[DEBUG] 2023-05-14 10:25:13.804 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@14b222aa: (connection evicted)
[DEBUG] 2023-05-14 10:25:13.898 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@61b244db: (connection evicted)
[DEBUG] 2023-05-14 10:25:13.987 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@64909d35: (connection evicted)
[DEBUG] 2023-05-14 10:25:15.236 [SpringApplicationShutdownHook] HikariPool - HikariPool-1 - After shutdown stats (total=0, active=0, idle=0, waiting=0)
[INFO ] 2023-05-14 10:25:15.236 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown completed.
[INFO ] 2023-05-14 10:26:40.607 [restartedMain] SpringBootRndApplication - Starting SpringBootRndApplication using Java 11.0.17 on tigerit-HP-ProDesk-600-G5-PCI-MT with PID 7751 (/home/tigerit/home_office/spring-boot-rnd/target/classes started by tigerit in /home/tigerit/home_office/spring-boot-rnd)
[DEBUG] 2023-05-14 10:26:40.608 [restartedMain] SpringBootRndApplication - Running with Spring Boot v2.7.0, Spring v5.3.20
[INFO ] 2023-05-14 10:26:40.608 [restartedMain] SpringBootRndApplication - No active profile set, falling back to 1 default profile: "default"
[DEBUG] 2023-05-14 10:26:41.537 [restartedMain] HikariConfig - HikariPool-1 - configuration:
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - allowPoolSuspension................................false
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - autoCommit................................true
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - catalog................................none
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - connectionInitSql................................none
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - connectionTestQuery................................none
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - connectionTimeout................................30000
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - dataSource................................none
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - dataSourceClassName................................none
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - dataSourceJNDI................................none
[DEBUG] 2023-05-14 10:26:41.538 [restartedMain] HikariConfig - dataSourceProperties................................{password=<masked>}
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - driverClassName................................none
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - exceptionOverrideClassName................................none
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - healthCheckProperties................................{}
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - healthCheckRegistry................................none
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - idleTimeout................................600000
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - initializationFailTimeout................................1
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - isolateInternalQueries................................false
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - jdbcUrl................................jdbc:postgresql://tiny.db.elephantsql.com:5432/rzumhkcr
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - keepaliveTime................................0
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - leakDetectionThreshold................................0
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - maxLifetime................................1800000
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - maximumPoolSize................................10
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - metricRegistry................................none
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - metricsTrackerFactory................................none
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - minimumIdle................................10
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - password................................<masked>
[DEBUG] 2023-05-14 10:26:41.539 [restartedMain] HikariConfig - poolName................................"HikariPool-1"
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - readOnly................................false
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - registerMbeans................................false
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - scheduledExecutor................................none
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - schema................................none
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - threadFactory................................internal
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - transactionIsolation................................default
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - username................................"rzumhkcr"
[DEBUG] 2023-05-14 10:26:41.540 [restartedMain] HikariConfig - validationTimeout................................5000
[INFO ] 2023-05-14 10:26:41.540 [restartedMain] HikariDataSource - HikariPool-1 - Starting...
[DEBUG] 2023-05-14 10:26:41.545 [restartedMain] DriverDataSource - Loaded driver with class name org.postgresql.Driver for jdbcUrl=jdbc:postgresql://tiny.db.elephantsql.com:5432/rzumhkcr
[DEBUG] 2023-05-14 10:26:44.971 [restartedMain] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@5cbf841
[INFO ] 2023-05-14 10:26:44.975 [restartedMain] HikariDataSource - HikariPool-1 - Start completed.
[DEBUG] 2023-05-14 10:26:45.075 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=1, active=1, idle=0, waiting=0)
[DEBUG] 2023-05-14 10:26:45.966 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@66688220
[DEBUG] 2023-05-14 10:26:46.870 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@68af609d
[DEBUG] 2023-05-14 10:26:47.477 [restartedMain] JwtRequestFilter - Filter 'jwtRequestFilter' configured for use
[DEBUG] 2023-05-14 10:26:47.773 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@b499436
[DEBUG] 2023-05-14 10:26:48.696 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@18f1a64a
[INFO ] 2023-05-14 10:26:48.735 [restartedMain] SpringBootRndApplication - Started SpringBootRndApplication in 8.322 seconds (JVM running for 8.83)
[DEBUG] 2023-05-14 10:26:49.237 [HikariPool-1 connection adder] PoolBase - HikariPool-1 - Failed to create/setup connection: FATAL: too many connections for role "rzumhkcr"
[DEBUG] 2023-05-14 10:26:49.237 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:26:50.057 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:26:50.946 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:26:52.054 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:26:53.433 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:26:55.288 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:26:57.708 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:01.107 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:05.893 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:13.212 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@41cccdc0
[DEBUG] 2023-05-14 10:27:13.758 [HikariPool-1 connection adder] PoolBase - HikariPool-1 - Failed to create/setup connection: FATAL: too many connections for role "rzumhkcr"
[DEBUG] 2023-05-14 10:27:13.759 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:14.973 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@4e20f4e8
[DEBUG] 2023-05-14 10:27:15.077 [HikariPool-1 housekeeper] HikariPool - HikariPool-1 - Pool stats (total=7, active=0, idle=7, waiting=0)
[DEBUG] 2023-05-14 10:27:15.553 [HikariPool-1 connection adder] PoolBase - HikariPool-1 - Failed to create/setup connection: FATAL: too many connections for role "rzumhkcr"
[DEBUG] 2023-05-14 10:27:15.553 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:16.335 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:17.622 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@69a58221
[DEBUG] 2023-05-14 10:27:18.188 [HikariPool-1 connection adder] PoolBase - HikariPool-1 - Failed to create/setup connection: FATAL: too many connections for role "rzumhkcr"
[DEBUG] 2023-05-14 10:27:18.188 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:18.989 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:19.925 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:21.083 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:22.453 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:24.268 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:26.721 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[DEBUG] 2023-05-14 10:27:30.126 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: too many connections for role "rzumhkcr"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2675) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2787) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:173) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:290) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.makeConnection(Driver.java:402) ~[postgresql-42.3.5.jar:42.3.5]
	at org.postgresql.Driver.connect(Driver.java:261) ~[postgresql-42.3.5.jar:42.3.5]
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726) ~[HikariCP-4.0.3.jar:?]
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712) ~[HikariCP-4.0.3.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:829) ~[?:?]
[INFO ] 2023-05-14 10:27:34.765 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown initiated...
[DEBUG] 2023-05-14 10:27:34.765 [SpringApplicationShutdownHook] HikariPool - HikariPool-1 - Before shutdown stats (total=8, active=0, idle=8, waiting=0)
[DEBUG] 2023-05-14 10:27:34.768 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@5cbf841: (connection evicted)
[DEBUG] 2023-05-14 10:27:34.771 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@66688220: (connection evicted)
[DEBUG] 2023-05-14 10:27:34.774 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@68af609d: (connection evicted)
[DEBUG] 2023-05-14 10:27:34.776 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@b499436: (connection evicted)
[DEBUG] 2023-05-14 10:27:34.864 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@18f1a64a: (connection evicted)
[DEBUG] 2023-05-14 10:27:34.956 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@41cccdc0: (connection evicted)
[DEBUG] 2023-05-14 10:27:35.044 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@4e20f4e8: (connection evicted)
[DEBUG] 2023-05-14 10:27:35.133 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@69a58221: (connection evicted)
[DEBUG] 2023-05-14 10:27:35.353 [HikariPool-1 connection adder] HikariPool - HikariPool-1 - Added connection org.postgresql.jdbc.PgConnection@69e58922
[DEBUG] 2023-05-14 10:27:35.353 [HikariPool-1 connection closer] PoolBase - HikariPool-1 - Closing connection org.postgresql.jdbc.PgConnection@69e58922: (connection evicted)
[DEBUG] 2023-05-14 10:27:35.447 [SpringApplicationShutdownHook] HikariPool - HikariPool-1 - After shutdown stats (total=0, active=0, idle=0, waiting=0)
[INFO ] 2023-05-14 10:27:35.447 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown completed.
[INFO ] 2023-05-14 10:28:48.647 [restartedMain] SpringBootRndApplication - Starting SpringBootRndApplication using Java 11.0.17 on tigerit-HP-ProDesk-600-G5-PCI-MT with PID 8070 (/home/tigerit/home_office/spring-boot-rnd/target/classes started by tigerit in /home/tigerit/home_office/spring-boot-rnd)
[INFO ] 2023-05-14 10:28:48.648 [restartedMain] SpringBootRndApplication - No active profile set, falling back to 1 default profile: "default"
[INFO ] 2023-05-14 10:28:49.593 [restartedMain] HikariDataSource - HikariPool-1 - Starting...
[INFO ] 2023-05-14 10:28:51.185 [restartedMain] HikariDataSource - HikariPool-1 - Start completed.
[INFO ] 2023-05-14 10:28:55.001 [restartedMain] SpringBootRndApplication - Started SpringBootRndApplication in 6.549 seconds (JVM running for 7.073)
[INFO ] 2023-05-14 10:29:30.287 [http-nio-8081-exec-1] JwtRequestFilter - not applying custom filter to: /health/status
[INFO ] 2023-05-14 10:29:33.578 [http-nio-8081-exec-2] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[WARN ] 2023-05-14 10:29:33.578 [http-nio-8081-exec-2] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:29:33.578 [http-nio-8081-exec-2] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:29:33.584 [http-nio-8081-exec-2] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:29:33.584 [http-nio-8081-exec-2] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:29:33.630 [http-nio-8081-exec-3] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-16x16.png
[WARN ] 2023-05-14 10:29:33.630 [http-nio-8081-exec-3] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:29:33.630 [http-nio-8081-exec-3] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:29:33.630 [http-nio-8081-exec-3] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:29:33.630 [http-nio-8081-exec-3] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:29:43.530 [http-nio-8081-exec-4] JwtRequestFilter - not applying custom filter to: /api/v1/auth/login
[INFO ] 2023-05-14 10:29:50.668 [http-nio-8081-exec-5] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[WARN ] 2023-05-14 10:29:50.668 [http-nio-8081-exec-5] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:29:50.668 [http-nio-8081-exec-5] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:29:50.668 [http-nio-8081-exec-5] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:29:50.668 [http-nio-8081-exec-5] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:29:50.707 [http-nio-8081-exec-6] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-16x16.png
[WARN ] 2023-05-14 10:29:50.707 [http-nio-8081-exec-6] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 10:29:50.707 [http-nio-8081-exec-6] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 10:29:50.707 [http-nio-8081-exec-6] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 10:29:50.707 [http-nio-8081-exec-6] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 10:29:59.467 [http-nio-8081-exec-7] JwtRequestFilter - doFilterInternal path: /api/v1/user
[INFO ] 2023-05-14 10:30:00.715 [http-nio-8081-exec-7] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhc2lmIiwiZXhwIjoxNjg0MDM4NzA0LCJ0eXBlIjoiYWNjZXNzIiwiaWF0IjoxNjg0MDM4NTg0fQ.uo47lExuKu0YwF1iAM993vRe9Io9T8Pu5Tg0Nk5Rx3WJpbb3ae8mLU5HlEDz5KeKzzlAOR3Ej8tcs1se4huE-g
[INFO ] 2023-05-14 10:30:00.735 [http-nio-8081-exec-7] JwtRequestFilter - Valid JWT Token In Header username: asif
[INFO ] 2023-05-14 10:30:00.735 [http-nio-8081-exec-7] JwtRequestFilter - token type: access
[WARN ] 2023-05-14 10:30:00.735 [http-nio-8081-exec-7] JwtRequestFilter - exceptionOccurred false
[INFO ] 2023-05-14 10:30:00.736 [http-nio-8081-exec-7] JwtRequestFilter - Handling request with access token
[INFO ] 2023-05-14 10:30:21.281 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown initiated...
[INFO ] 2023-05-14 10:30:23.778 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown completed.
[INFO ] 2023-05-14 23:23:32.365 [restartedMain] SpringBootRndApplication - Starting SpringBootRndApplication using Java 11.0.8 on Asifs-MacBook-Pro.local with PID 6191 (/Users/asif/My_Mac_New/Spring Boot/spring-boot-rnd/target/classes started by asif in /Users/asif/My_Mac_New/Spring Boot/spring-boot-rnd)
[INFO ] 2023-05-14 23:23:32.369 [restartedMain] SpringBootRndApplication - No active profile set, falling back to 1 default profile: "default"
[INFO ] 2023-05-14 23:23:35.453 [restartedMain] HikariDataSource - HikariPool-1 - Starting...
[INFO ] 2023-05-14 23:23:37.087 [restartedMain] HikariDataSource - HikariPool-1 - Start completed.
[INFO ] 2023-05-14 23:23:47.591 [restartedMain] SpringBootRndApplication - Started SpringBootRndApplication in 15.609 seconds (JVM running for 22.242)
[INFO ] 2023-05-14 23:25:23.981 [http-nio-8081-exec-1] JwtRequestFilter - not applying custom filter to: /
[INFO ] 2023-05-14 23:25:24.296 [http-nio-8081-exec-2] JwtRequestFilter - doFilterInternal path: /swagger-ui/index.html
[WARN ] 2023-05-14 23:25:24.296 [http-nio-8081-exec-2] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 23:25:24.296 [http-nio-8081-exec-2] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 23:25:24.321 [http-nio-8081-exec-2] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 23:25:24.321 [http-nio-8081-exec-2] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 23:25:24.976 [http-nio-8081-exec-3] JwtRequestFilter - not applying custom filter to: /v3/api-docs/swagger-config
[INFO ] 2023-05-14 23:25:25.111 [http-nio-8081-exec-4] JwtRequestFilter - doFilterInternal path: /apple-touch-icon-precomposed.png
[WARN ] 2023-05-14 23:25:25.111 [http-nio-8081-exec-4] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 23:25:25.111 [http-nio-8081-exec-4] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 23:25:25.112 [http-nio-8081-exec-4] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 23:25:25.112 [http-nio-8081-exec-4] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 23:25:25.179 [http-nio-8081-exec-5] JwtRequestFilter - not applying custom filter to: /v3/api-docs
[INFO ] 2023-05-14 23:25:25.326 [http-nio-8081-exec-6] JwtRequestFilter - doFilterInternal path: /apple-touch-icon.png
[WARN ] 2023-05-14 23:25:25.326 [http-nio-8081-exec-6] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 23:25:25.326 [http-nio-8081-exec-6] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 23:25:25.326 [http-nio-8081-exec-6] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 23:25:25.326 [http-nio-8081-exec-6] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-7] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-32x32.png
[INFO ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-8] JwtRequestFilter - doFilterInternal path: /swagger-ui/favicon-16x16.png
[WARN ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-7] JwtRequestFilter - No JWT Token In Header
[WARN ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-8] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-7] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[INFO ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-8] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-8] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-7] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-8] JwtRequestFilter - exceptionOccurred true
[WARN ] 2023-05-14 23:25:25.349 [http-nio-8081-exec-7] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 23:25:31.170 [http-nio-8081-exec-9] JwtRequestFilter - not applying custom filter to: /health/status
[INFO ] 2023-05-14 23:30:08.935 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown initiated...
[INFO ] 2023-05-14 23:30:11.507 [SpringApplicationShutdownHook] HikariDataSource - HikariPool-1 - Shutdown completed.
[INFO ] 2023-05-14 23:32:05.543 [restartedMain] SpringBootRndApplication - Starting SpringBootRndApplication using Java 11.0.8 on Asifs-MacBook-Pro.local with PID 6303 (/Users/asif/My_Mac_New/Spring Boot/spring-boot-rnd/target/classes started by asif in /Users/asif/My_Mac_New/Spring Boot/spring-boot-rnd)
[INFO ] 2023-05-14 23:32:05.546 [restartedMain] SpringBootRndApplication - No active profile set, falling back to 1 default profile: "default"
[INFO ] 2023-05-14 23:32:07.814 [restartedMain] HikariDataSource - HikariPool-1 - Starting...
[INFO ] 2023-05-14 23:32:09.263 [restartedMain] HikariDataSource - HikariPool-1 - Start completed.
[INFO ] 2023-05-14 23:32:16.075 [restartedMain] SpringBootRndApplication - Started SpringBootRndApplication in 10.906 seconds (JVM running for 17.263)
[INFO ] 2023-05-14 23:32:34.643 [http-nio-8081-exec-1] JwtRequestFilter - doFilterInternal path: /swagger-ui/index.html
[WARN ] 2023-05-14 23:32:34.643 [http-nio-8081-exec-1] JwtRequestFilter - No JWT Token In Header
[INFO ] 2023-05-14 23:32:34.644 [http-nio-8081-exec-1] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> null
[WARN ] 2023-05-14 23:32:34.660 [http-nio-8081-exec-1] JwtRequestFilter - Unable to get JWT Token or Token is blacklisted
[WARN ] 2023-05-14 23:32:34.661 [http-nio-8081-exec-1] JwtRequestFilter - exceptionOccurred true
[INFO ] 2023-05-14 23:32:35.047 [http-nio-8081-exec-2] JwtRequestFilter - not applying custom filter to: /v3/api-docs/swagger-config
[INFO ] 2023-05-14 23:32:35.135 [http-nio-8081-exec-3] JwtRequestFilter - not applying custom filter to: /v3/api-docs
[INFO ] 2023-05-14 23:32:38.461 [http-nio-8081-exec-4] JwtRequestFilter - not applying custom filter to: /health/status
[INFO ] 2023-05-14 23:33:01.857 [http-nio-8081-exec-5] JwtRequestFilter - not applying custom filter to: /api/v1/auth/login
[INFO ] 2023-05-14 23:34:33.597 [http-nio-8081-exec-7] JwtRequestFilter - doFilterInternal path: /api/v1/user
[INFO ] 2023-05-14 23:34:39.755 [http-nio-8081-exec-7] JwtRequestFilter - JwtRequestFilter: doFilterInternal jwtToken-> eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhc2lmIiwiZXhwIjoxNjg0MDg1NzAyLCJ0eXBlIjoiYWNjZXNzIiwiaWF0IjoxNjg0MDg1NTgyfQ.N3diWyZZ20eQcU50g8CTiT9TANHvuy7I2FWYUhClO5rSzdJkCFjPjd-Xqz4gh1x6QpJvITFz18IkSoykLT_wyQ
[INFO ] 2023-05-14 23:34:39.765 [http-nio-8081-exec-7] JwtRequestFilter - Valid JWT Token In Header username: asif
[INFO ] 2023-05-14 23:34:39.765 [http-nio-8081-exec-7] JwtRequestFilter - token type: access
[WARN ] 2023-05-14 23:34:39.765 [http-nio-8081-exec-7] JwtRequestFilter - exceptionOccurred false
[INFO ] 2023-05-14 23:34:39.765 [http-nio-8081-exec-7] JwtRequestFilter - Handling request with access token
